engine/.rubocop.yml

```diff
@@ -0,0 +1,28 @@
+inherit_gem:
+  citizens-advice-style:
+    - default.yml
+    - default-rails.yml
+
+AllCops:
+  SuggestExtensions: false
+  Exclude:
```

Comment:
if you remove these lines does it fail? Quick way to test would be just running locally so you don't need to push up